index.html

```diff
@@ -1,31 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<meta charset="UTF-8">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<link rel="stylesheet" href="./CSS/style.css">
-	<title>Document</title>
-</head>
-<body>
-	<header class="header">
-		<h1>Awesome books</h1>
-	</header>
-	<main class="main">
-		<section class="list-section">
-		</section>
-		<form>
-			<label for="title">
-				<input type="text" id="title" placeholder="Title" required>
-			</label>
-			<label for="author">
-				<input type="text" id="author" placeholder="Author" required>
-			</label>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./CSS/style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h3>Awesome Books</h3>
+        <ul>
+          <li class="listBtn">List</li>
+          
+          <li class="listBtn">Add New</li>
+          
+          <li class="listBtn">Contact</li>
+        </ul>
+      </nav>
+	<div class="date"></div>
+    </header>
 
-			<button type="submit" class="btn-submit">add</button>
-		</form>
-	</main>
-	
-	<script src="./JS/main.js"></script>
-</body>
-</html>
\ No newline at end of file
+    <main class="main">
+      <section class="sec welcome active">
+        <h2 class="headTitles">Awesome books</h2>
+
+        <section class="list-section"></section>
+      </section>
+
+      <section class="sec addNew">
+        <form>
+          <h2 class="headTitles">Add a new Book</h2>
+          <label for="title">
+            <input type="text" id="title" placeholder="Title" required />
+          </label>
+          <label for="author">
+            <input type="text" id="author" placeholder="Author" required />
+          </label>
```

Comment:
- [ ] [OPTIONAL] After I input and submit the book its adding to the list but this alert appearing. Please kindly fix this issue 👍

![image](https://user-images.githubusercontent.com/69011963/153607261-8f73b05c-8f6e-445c-89e3-51485cc19190.png)